src/auth/token.service.ts

```diff
@@ -119,7 +119,7 @@ export class TokenService {
       this.logger.error(err);
       return {
         isValid: false,
-        errorMessage: 'Refresh token is invalid',
+        errorMessage: 'Token refresh no es valido',
```

Comment:
Maybe it should be 'El refresh token no es válido'?

Reply:
Message modified in last commit